src/hooks/useLocation.js

```diff
@@ -24,24 +23,29 @@ const useLocation = () => {
   const [location, setLocation] = useState(INITIAL_COORDS);
 
   useEffect(() => {
+    let unsubscribe = null;
     async function requestLocation() {
       try {
         let granted = await RNLocation.checkPermission(PERMISSIONS);
         if (!granted) {
           granted = await RNLocation.requestPermission(PERMISSIONS);
         }
         if (granted) {
-          const { latitude, longitude } = await RNLocation.getLatestLocation({ timeout: TIMEOUT });
-          setLocation({ latitude, longitude });
+          unsubscribe = RNLocation.subscribeToLocationUpdates(location => {
+            const { latitude } = location[location.length - 1];
+            const { longitude } = location[location.length - 1];
+            setLocation({ latitude, longitude });
+          });
         }
       } catch (error) {
         throw parseError(error);
       }
     }
     requestLocation();
+    return () => unsubscribe();
```

Comment:
There is no need to create a new function just to invoke a function that exists already
```suggestion
    return unsubscribe;
```